draft-irtf-cfrg-hpke.md

```diff
@@ -375,133 +375,126 @@ and {{encryption-context}} for details.
 
 ## DH-Based KEM {#dhkem}
 
-Suppose we are given a KDF, and the following interface to a cyclic group `G` (and hash 
-function `Hash`). Implementations of this interface based on the P-256, P-384 and the
-P-521 family of NIST curves as well as the Edwards25519 and Edwards448 curves can be 
-found in {{ciphersuites}}.
+Suppose we are given a KDF, and the following interface to a cyclic group `G`. Implementations of this interface based on the P-256, P-384 and the
+P-521 family of NIST curves as well as the Edwards25519 and Edwards448 curves are described in in {{ciphersuites}}.
 
 * Data Types:
-  - element of field `F` : finite field of "scalars".
-  - element of group `G` : additive group of "points" over `F`.
+  - element of field `F` : finite field of “scalars”.
+  - element of group `G` : additive group of “points” over `F`.
 
 * Constants:
   - `B` : generator of group `G`
-  - `2n` : length in octets (rounded to nearest even integer) of encoding of a field
-  element in `F`
-  - `ptLen` : length, in octets, of a point encoded as an octet string
-  - `hLen` : length, in octets, of hash function's output
+  - `Nf` : length in bytes (rounded to nearest even integer) of a field
+    element in `F` encoded as a byte string
+  - `Np` : length in bytes of a point encoded as a byte string
 
 * Arithmetic Operators:
   - `+` : point/point addition : `G x G -> G`
   - `.` : scalar/point multiplication : `F x G -> G`
   - `++` : scalar/scalar addition : `F x F -> F`
   - `**` : scalar/scalar multiplication : `F x F -> F`
-  - `||` : string concatenation
 
 * Type Conversion Algorithms:
-  - `int_to_string(a_c, len)` : converts scalar `a` to string of `len` octets
-  - `string_to_int(a_s)` : converts an octet string `a_string` to scalar
-  - `point_to_string(a)` : converts point to string of `ptLen` octets
-  - `string_to_point(a_s)` : converts string of `ptLen` octets to point 
```

Comment:
I suggest calling them SerializePoint and DeserializePoint to align with the other parts of the RFC.